ministryofjustice/analytics-platform-control-panel
# Feature/local kube connection anpl 857

This PR covers a spike where we started work on a local kind cluster and control panel instance for development purposes. It is not yet complete, so this PR should not be merged. However, it is partially wrapped up in that everything in this PR so far has been documented in `current_status_of_spike.md`.

## What

- Control Panel can now communicate with a kind cluster.
- A user can be authenticated, their namespace is created but the `provision user` chart cannot yet be run.
- Tools to deploy are listed and a deployment can be requested, but the deployment itself does not yet work.

## How to review

1. Read `current_status_of_spike.md`
2. Check that the descriptions in there cover all changes to the code in this PR
3. Try out the steps in that doc to create your own local kind cluster and control panel instance

See #ANPL-857.
